### PR TITLE
allow symfony 5, drop symfony 2 and php 5

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,28 +1,37 @@
 language: php
 
 php:
-  - 5.6
   - 7.0
```
Comment (Tobion) [inline .travis.yml:4]: 7.0 (and 7.1) are not supported by php anymore

**Comment by Tobion** [comment]
Why is there a composer.lock file in a library? Shouldn't that be excluded?

File: .travis.yml (hunk 1 above)
Comment (dbu) [reply .travis.yml:4]: yeah, but some people are stuck on outdated infrastructure. as long as its not creating problems, i would prefer to allow older versions.

**Comment by dbu** [comment]
afaik @ChristianRiesen wanted that in the commit because there is a system to build a .phar from this code. with the .lock file, we avoid people ending up with a .phar that uses versions of the dependencies that have not been tested. (then again, there are so few tests that i don't know if its worth it)

**Comment by dbu** [comment]
created #48 about the composer.lock file and #47 for minimum version

**Pull request merged**